### PR TITLE
Add more number formatting for AE2

### PR DESCRIPTION
**Pull request by Connor-Colenso**
The formatting crusade continues...

### CODE REVIEW

**Comment by repo-alt** [comment]
Can you please use ReadableNumberConverter instead? (and maybe remove `formatNumbers`)

File: src/main/java/appeng/client/gui/implementations/GuiCraftConfirm.java [hunk 1]
```diff
@@ -210,6 +210,8 @@ private boolean isSimulation() {
         return ((ContainerCraftConfirm) this.inventorySlots).isSimulation();
     }
 
+    final ReadableNumberConverter converter = ReadableNumberConverter.INSTANCE;
```
Comment (repo-alt) [inline src/main/java/appeng/client/gui/implementations/GuiCraftConfirm.java:213]: This seems to be redundant if you use java.text.NumberFormat directly

Comment (Connor-Colenso) [reply src/main/java/appeng/client/gui/implementations/GuiCraftConfirm.java:213]: I would concur but for some reason removing it and using NumberFormat directly caused the game to crash. It was written like this before.

Comment (repo-alt) [reply src/main/java/appeng/client/gui/implementations/GuiCraftConfirm.java:213]: Really? What is in the crash log?

**Review by repo-alt — APPROVED** [review]

**Pull request merged**